utils/keycodes.js

```diff
@@ -7,3 +7,4 @@ export const UP = 38;
 export const RIGHT = 39;
 export const DOWN = 40;
 export const DELETE = 46;
+export const LETTER_A = 65;
```

Comment:
I recall discussions about compatibility issues for `event.key` in some browsers. cc @aduth 
And we're using `keyCode` everywhere now.